Playground/RxSwiftExtPlayground.playground/Pages/Index.xcplaygroundpage/Contents.swift

```diff
@@ -21,6 +21,7 @@
  
  ## Index:
  
+ - [apply()](apply) operator, takes a transform function and applies it to the sequence, returning the sequence returned by the transform.
```

Comment:
I think "returning the sequence returned by the transform" perhaps is a bit confusing/mouthful ? 
What are your thoughts about just "returning the transformed sequence" ?